src/applications/mhv-secure-messaging/tests/containers/ThreadDetails.unit.spec.jsx

```diff
@@ -760,4 +761,53 @@ describe('Thread Details container', () => {
       'Your account is no longer connected to SM_TO_VA_GOV_TRIAGE_GROUP_TEST',
     );
   });
+
+  it('does not load if recipients.associatedTriageGroupsQty is undefined', async () => {
+    const state = {
+      sm: {
+        folders: {
+          folder: inbox,
+        },
+        threadDetails,
+        recipients: {
+          allRecipients: [],
+          allowedRecipients: [],
+          blockedRecipients: [],
+          associatedTriageGroupsQty: undefined,
+          associatedBlockedTriageGroupsQty: undefined,
+          noAssociations: undefined,
+          allTriageGroupsBlocked: undefined,
+        },
+      },
+      drupalStaticData: {
+        vamcEhrData: {
+          data: {
+            ehrDataByVhaId: [
+              {
+                facilityId: '662',
+                isCerner: false,
+              },
+              {
+                facilityId: '636',
+                isCerner: false,
+              },
+            ],
+          },
+        },
+      },
+      featureToggles: {},
+    };
+
+    const retrieveMessageThreadSpy = sinon.spy(
+      messagesActions,
+      'retrieveMessageThread',
+    );
+
+    setup(state);
+
+    await waitFor(() => {
+      expect(retrieveMessageThreadSpy.calledOnce).not.to.be.true;
+      retrieveMessageThreadSpy.restore();
+    });
```

Comment:
this test may need to be revised. I was trying to break it by adding those redux values and it was still passing